apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-sidebar/index.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import LaunchMenu from '../launch-menu';
+import './styles.scss';
+
+const LaunchSidebar = () => {
+	return (
+		<div className="nux-launch-sidebar">
+			<h2>{ __( "You're almost there!", 'full-site-editing' ) }</h2>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'You're almost there!' )` *ES Score: 13*